### PR TITLE
[IMP] multi-company support

### PR DESCRIPTION
**Pull request by brian10048**
@max3903 Please review for #162 

I followed your spec.  I'm wondering if we should we also setup multi-company support on the FSM category and FSM order template models?

### CODE REVIEW

**Review by osi-scampbell — APPROVED** [review]
Everything looks good! If I had to make one suggestion it would be on fsm_territory. Here we have the company_id field in the right column but on fsm_region, fsm_district, and fsm_branch the company_id field is in the left column. For consistency purposes should we try to keep that field in the same column of each form view?

**Comment by brian10048** [comment]
I agree. I modified to keep consistency with other modules of Odoo and moved company fields to right hand column in views

**Review by max3903 — APPROVED** [review]

**Review by murtuzasaleh — APPROVED** [review]

File: fieldservice/security/company_security.xml [hunk 1]
```diff
@@ -0,0 +1,73 @@
+<odoo noupdate="1">
```
Comment (murtuzasaleh) [inline fieldservice/security/company_security.xml:2]: Please add this line <?xml version="1.0" encoding="utf-8"? >

File: fieldservice_vehicle/security/vehicle_security.xml [hunk 2]
```diff
@@ -0,0 +1,10 @@
+<odoo noupdate="1">
```
Comment (murtuzasaleh) [inline fieldservice_vehicle/security/vehicle_security.xml:2]: Please add this line <?xml version="1.0" encoding="utf-8"? >

**Comment by murtuzasaleh** [comment]
LGTM @max3903 Can you please merge this PR.

**Pull request merged**